src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -152,7 +152,7 @@ internal string GetDebuggerDisplay()
                         }
 
                         string identifierValueText = usingDirective.Alias.Name.Identifier.ValueText;
-                        SourceMemberContainerTypeSymbol.ReportTypeNamedRecord(identifierValueText, compilation, diagnostics, usingDirective.Alias.Name.Location);
+                        SourceMemberContainerTypeSymbol.ReportTypeNamedRecord(usingDirective.Alias.Name.Identifier.Text, compilation, diagnostics, usingDirective.Alias.Name.Location);
 
                         if (usingAliases != null && usingAliases.ContainsKey(identifierValueText))
```

Comment:
Consider moving the `identifierValueText` declaration closer to its usage to make things more clear.